From 5606b510e79a09fe64edfab6b8ab66bb9bc7f9eb Mon Sep 17 00:00:00 2001
Message-ID: <5606b510e79a09fe64edfab6b8ab66bb9bc7f9eb.1721514686.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1721514686.git.sharlatanus@gmail.com>
References: <cover.1721514686.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 Jul 2024 20:15:36 +0100
Subject: [PATCH 24/34] gnu: python-fast-histogram: Adjust inputs.

* gnu/packages/python-science.scm (python-fast-histogram): Fix lint
warnings and adjust inputs.
[native-inputs]: Add python-setuptools-scm.

Change-Id: Ic51660d7542114f2bb45be4720ab4d899633e56d
---
 gnu/packages/python-science.scm | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/python-science.scm b/gnu/packages/python-science.scm
index 8ba3432637..7012a942d3 100644
--- a/gnu/packages/python-science.scm
+++ b/gnu/packages/python-science.scm
@@ -1551,17 +1551,20 @@ (define-public python-fast-histogram
             (lambda _
               (invoke "python" "setup.py" "build_ext" "--inplace"))))))
     (propagated-inputs (list python-numpy))
-    (native-inputs (list python-hypothesis python-pytest))
+    (native-inputs
+     (list python-hypothesis
+           python-pytest
+           python-setuptools-scm))
     (home-page "https://github.com/astrofrog/fast-histogram")
     (synopsis "Fast simple 1D and 2D histograms")
     (description
      "The fast-histogram mini-package aims to provide simple and fast
-histogram functions for regular bins that don't compromise on performance. It
+histogram functions for regular bins that don't compromise on performance.  It
 doesn't do anything complicated - it just implements a simple histogram
-algorithm in C and keeps it simple. The aim is to have functions that are fast
-but also robust and reliable. The result is a 1D histogram function here that
-is 7-15x faster than @code{numpy.histogram}, and a 2D histogram function that
-is 20-25x faster than @code{numpy.histogram2d}.")
+algorithm in C and keeps it simple.  The aim is to have functions that are
+fast but also robust and reliable.  The result is a 1D histogram function here
+that is 7-15x faster than @code{numpy.histogram}, and a 2D histogram function
+that is 20-25x faster than @code{numpy.histogram2d}.")
     (license license:bsd-3)))
 
 (define-public python-fastcluster
-- 
2.41.0

